### PR TITLE
Pt 166927306 names as strings

### PR DESCRIPTION
**Pull request by gorillainduction**
https://www.pivotaltracker.com/story/show/166927306

### CODE REVIEW

File: apps/aecontract/src/aect_contracts.erl [hunk 1]
```diff
@@ -145,14 +148,21 @@ is_legal_version_in_protocol(create, #{vm := ?VM_AEVM_SOPHIA_2, abi := ?ABI_AEVM
...
         ?FORTUNA_PROTOCOL_VSN -> true;
-        ?LIMA_PROTOCOL_VSN    -> true %% TODO: If you bump to VM_AEVM_SOPHIA_3 please turn this off!
+        ?LIMA_PROTOCOL_VSN    -> false
+    end;
+is_legal_version_in_protocol(create, #{vm := ?VM_AEVM_SOPHIA_4, abi := ?ABI_AEVM_SOPHIA_1}, ProtocolVersion) ->
+    case ProtocolVersion of
+        ?ROMA_PROTOCOL_VSN    -> false;
+        ?MINERVA_PROTOCOL_VSN -> false;
+        ?FORTUNA_PROTOCOL_VSN -> false;
+        ?LIMA_PROTOCOL_VSN    -> true %% TODO: If you bump to VM_AEVM_SOPHIA_4 please turn this off!
```
Comment (hanssv) [inline apps/aecontract/src/aect_contracts.erl]: The comment is symmetric to the previous code, but I am not sure what it means?

**Comment by hanssv** [comment]
Needs a rebase...

File: apps/aecontract/src/aect_contracts.erl (hunk 1 above)
Comment (gorillainduction) [reply apps/aecontract/src/aect_contracts.erl]: It i a bit cryptic, I agree. It was originally intended as a reminder to turn off creation of contracts with old VM:s but it doesn't read well.

Comment (hanssv) [reply apps/aecontract/src/aect_contracts.erl]: Shouldn't it at least say `SOPHIA_5` then?

**Pull request merged**